### PR TITLE
Initial Controller

### LINKED ISSUE #3: Controller
We need a motor controller for our sub. One possibility is to use a completely third party implementation like [ArduSub](https://www.ardusub.com/). This task however is focused on developing a controller in ROS that we can run on the same computer as the rest of the software. Additionally, having our own controller would make future iteration easier, and the software stack overall would be more cohesive. Future controller sub-tasks will be created as more targeted issues are identified.

### PR DESCRIPTION
**Pull request by chachmu**
Sets up some simple controller nodes:
1. **thruster_controller:** Takes in a commanded body acceleration on `/cmd_accel` and uses the thruster positions published on `/tf` to calculate individual thruster commands and publish them on `/pontus/thruster_<id>/cmd_thrust`. I just hard coded an approximation for the vehicle's mass moments of inertia in the node for now but eventually we will need more accurate numbers.

2. **velocity_controller:** Takes in a commanded body velocity on `/cmd_vel` and uses PID to generate body acceleration commands on `/cmd_accel`. Currently it is just set to use a P controller with a coefficient of 1 which doesn't perform particularly well so we will need to tune the PID values at some point.

Make sure to merge #9 first since this depends on it. Closes #3 although we might want to open a new issue for improving the controller through cleaning up the code, tuning PID, and adding additional features/parameters

I'll write some more documentation in the readme tomorrow but here are the basic steps for testing:
1. Launch the simulator using `ros2 launch pontus_sim sim.py`
2. Get odometry data from either gazebo or robot_localization:
    * robot_localization: `ros2 launch pontus_localization localization.py`. This is the most realistic but also has some pretty serious IMU drift.
    * gazebo: `ros2 launch pontus_sim odom_bridge.py`
3. Spawn the vehicle in the simulator: `ros2 launch pontus_spawn.py`
4. Run the control nodes: `ros2 launch pontus_controller vel_control.py`
5. By default the vehicle will try to maintain a velocity of 0 (currently it doesn't do a very good job due to the PID not being tuned) or you can publish body velocity messages in Twist format to `cmd_vel`.

### CODE REVIEW

**Comment by chachmu** [comment]
Ok I upped the P value to 2 and added an I value of 1 to the linear velocities and now it can hold itself stable although it's still pretty slow to stabilize so we probably need to crank P up a lot. 

I also added a new `pontus_bringup` package that we can use for top level launch files to start all the nodes for simulation testing or competition. Currently there are only 2 launch files in it:

1. simulation.py: Launches the simulation and robot_localization, spawns the vehicle, and runs the velocity controller and thruster controllers.
2. odom_simulation.py: Does the same as above but instead of using robot_localization it uses the published gazebo odometry.

Both launch files also accept a `world` argument for which gazebo world to launch (currently have `underwater.world` and `sensor.world`) and a `static` boolean argument that locks the vehicle in place so it doesn't float around.

Long story short if you just want to run the simulator to test some stuff right now your best bet is to just use `ros2 launch pontus_bringup odom_simulation.py`.

**Review by mroglan — APPROVED** [review]
